terraform/python/eks/main.tf

```diff
@@ -263,6 +263,45 @@ resource "kubernetes_service" "python_r_app_service" {
   }
 }
 
+resource "kubernetes_ingress_v1" "python-r-app-ingress" {
```

Comment:
Why did we not have this before? Is this needed? Do we do the same for Java?

Reply:
It's only needed to check if the remote sample application endpoint is available. Yes, we do the same for Java